tests/e2e/specs/features/02_invitations.cy.js

```diff
@@ -38,7 +38,7 @@ describe("Invitation feature", () => {
     cy.task("get-user", "user0").then((user) => cy.login(user));
     cy.visit("/invitations");
 
-    cy.hook("btn-accept-all-invit").click();
+    cy.hook("btn-accept-all-invit", { timeout: 60000 }).click();
```

Comment:
Is this timeout really useful ?
This test used to pass without problem before and it doesn't like your modifications have impacted that part of the code.